### ISSUE TITLE
Set up GitHub Actions

### ISSUE THREAD

**Issue by haneujeen** [haneujeen/cease#2]

**Comment by sykr1313** [comment]
GitHub 액션을 통해 배포를 하면 도커 컨테이너가 자동으로 생성이 되나요? GitHub 액션 사용 경험이 없어서 물어봅니다

**Comment by haneujeen** [comment]
> GitHub 액션을 통해 배포를 하면 도커 컨테이너가 자동으로 생성이 되나요? GitHub 액션 사용 경험이 없어서 물어봅니다

네, GitHub Actions 워크플로를 통해 도커 이미지를 생성하고 레지스트리에 등록할 수 있습니다. 이렇게 빌드된 이미지를 Azure 또는 다른 클라우드 서비스에 배포하는 워크플로를 GitHub Actions 워크플로에 추가할 수 있습니다. 
GitHub이 호스팅하는 무료 runner 가상 머신 내에서 컨테이너를 실행하는 워크플로를 작성하는 것은 가능하나, CI/CD 파이프라인 구축 마일스톤과는 별개의 이슈이기 때문에 무시하셔도 됩니다
